test/lib/next-test-utils.js

```diff
@@ -669,16 +669,27 @@ export async function hasRedbox(browser, expected = true) {
 
 export async function getRedboxHeader(browser) {
   return retry(
-    () =>
-      evaluate(browser, () => {
+    () => {
+      if (shouldRunTurboDevTest()) {
         const portal = [].slice
           .call(document.querySelectorAll('nextjs-portal'))
           .find((p) =>
-            p.shadowRoot.querySelector('[data-nextjs-dialog-header]')
+            p.shadowRoot.querySelector('[data-nextjs-turbo-dialog-body]')
```

Comment:
Not sure what's going on, but if I try to dynamically assign query str like

```
const query = shouldRunTurboDevTest()? 'x' : 'y'

...

p.shadowRoot.querySelector(query)

```

Then bunch of test started to fail. It looks like fn is passed into evaluation into brower but lost const param's value somehow.